### PR TITLE
Fix UI :-  Glossary default item after clicking from list to navigation

### LINKED ISSUE #7878: Glossary Items issue after click head navigation from glossary item
**Affected module**
Glossary selected item

**Describe the bug**
After click header glossary navigation from glossary child item/ thread inappropriate data coming.

**To Reproduce**

Screenshots or steps to reproduce

https://user-images.githubusercontent.com/66266464/193546569-610acbc6-1886-4538-9ad9-1c40f832f61e.mov

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes :

- Issue : Closes #7878
- Fix glossary default item after clicking from list to navigation
- Increase font weight of tags list as per glossary

#
### Type of change :

- [x] Bug fix
- [x] Improvement

#
### Frontend Preview (Screenshots) :
Issue : 

https://user-images.githubusercontent.com/66266464/193534805-7510d20e-53a4-4a67-a1c2-3f5aa1f25500.mov

Resolved : 
 

https://user-images.githubusercontent.com/66266464/193534839-4361e749-66c2-4205-965c-176edd786dc6.mov

image
image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
 @open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/pages/GlossaryPage/GlossaryPageV1.component.tsx [hunk 1]
```diff
@@ -287,9 +287,10 @@ const GlossaryPageV1 = () => {
   ) => {
     handleChildLoading(true);
     const hierarchy = getTermPosFromGlossaries(arrGlossary, dataFQN);
+    const isFQN = dataFQN;
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/pages/GlossaryPage/GlossaryPageV1.component.tsx:290]: do we need this variable? can't we use dataFQN directly?

**Comment by Sachin-chaurasiya** [comment]
@Ashish8689 , Can you create the issue and describe there what was the issue that we are fixing in this PR?

File: openmetadata-ui/src/main/resources/ui/src/pages/GlossaryPage/GlossaryPageV1.component.tsx (hunk 1 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/pages/GlossaryPage/GlossaryPageV1.component.tsx]: @ShaileshParmar11  changes updated!.

File: openmetadata-ui/src/main/resources/ui/src/pages/GlossaryPage/GlossaryPageV1.component.tsx [hunk 2]
```diff
@@ -288,8 +288,8 @@ const GlossaryPageV1 = () => {
     handleChildLoading(true);
     const hierarchy = getTermPosFromGlossaries(arrGlossary, dataFQN);
     if (hierarchy.length < 2) {
-      setSelectedData(arrGlossary[hierarchy[0]]);
-      handleSelectedKey(dataFQN);
+      setSelectedData(dataFQN ? arrGlossary[hierarchy[0]] : arrGlossary[0]);
+      handleSelectedKey(dataFQN ? dataFQN : arrGlossary[0].name);
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/pages/GlossaryPage/GlossaryPageV1.component.tsx:292]:
```suggestion
      handleSelectedKey(dataFQN ? dataFQN : arrGlossary[0]?.name);
```

**Review by ShaileshParmar11 — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/src/pages/GlossaryPage/GlossaryPageV1.component.tsx (hunk 2 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/pages/GlossaryPage/GlossaryPageV1.component.tsx]: @Sachin-chaurasiya  Updated!

**Review by Sachin-chaurasiya — APPROVED** [review]

**Comment by Sachin-chaurasiya** [comment]
@Ashish8689 , Merge main into your branch.

**Pull request merged**